### PR TITLE
Resolved issue with is_lcp_statically_discoverable

### PR DESCRIPTION
**Pull request by kevinfarrugia**
The raw HTML (from response_body) uses relative URLs for images, for example "/static/img/logo.png", while calling .src on a document with a location, gives you absolute URLs, such as https://happymod.com/static/img/logo.png.

Consequently, when checking for the LCP element, we need to wrap the src with new URL to get an absolute URL. 

---
**Test websites**:

- https://happymod.com/
- https://httparchive.org/

### CODE REVIEW

File: dist/performance.js [hunk 1]
```diff
@@ -306,7 +306,7 @@ function getRawLcpElement(rawDoc, lcpUrl) {
             src = i.getAttribute('href');
         }
 
-        return src == lcpUrl;
+        return new URL(src, location.href).href == lcpUrl;
```
Comment (rviscomi) [inline dist/performance.js:309]: Can you also wrap the srcset check above in new URL?

> src = splitSrcSet(i.srcset).find(src => src == lcpUrl);

**Comment by tunetheweb** [comment]
@kevinfarrugia , @rviscomi is this good to merge?

**Comment by rviscomi** [comment]
The change LGTM, I'm just not sure why the GH action isn't testing the URLs in the description

**Review by rviscomi — APPROVED** [review]

**Comment by max-ostapenko** [comment]
> The change LGTM, I'm just not sure why the GH action isn't testing the URLs in the description

`Test websites:` must be bold.

**Pull request merged**